src/components/Bookmark.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+
+function Bookmark() {
+  const [bookmarkNotActiv, setBookmarkActiv] = useState(false)
+
+  return (
+    <StyledButton
+      onClick={() => {
+        setBookmarkActiv(!bookmarkNotActiv)
+      }}
+    >
+      {!bookmarkNotActiv ? (
```

Comment:
I'm so so so sorry, but you're missing an -e here. 
```suggestion
  const [bookmarkNotActive, setBookmarkActive] = useState(false)

  return (
    <StyledButton
      onClick={() => {
        setBookmarkActive(!bookmarkNotActive)
      }}
    >
      {!bookmarkNotActive ? (
```